phpBB/includes/functions.php

```diff
@@ -4969,6 +4969,8 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 		}
 	}
 
+	$user->update_session_infos();
```

Comment:
Ok, I guess we'll have to prevent session refresh (if wanted) using the session time then. I guess that's fine, too.